Example/NYTPhotoViewer-Swift/PhotosProvider.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PhotosProvider.swift
+//  NYTPhotoViewer
+//
+//  Created by Mark Keefe on 3/20/15.
+//  Copyright (c) 2015 The New York Times. All rights reserved.
+//
+
+import UIKit
+
+/**
+*   In Swift 1.2, the following file level constants can be moved inside the class for better encapsulation
+*/
+let CustomEverythingPhotoIndex = 1, DefaultLoadingSpinnerPhotoIndex = 3, NoReferenceViewPhotoIndex = 4
+let PrimaryImageName = "NYTimesBuilding"
```

Comment:
In this case, its used repeatedly, so relying on the variable and code completion keeps me from entering a typo if nothing else.